cypress/integration/common/index.js

```diff
@@ -22,6 +22,13 @@ When('the user navigates to {string}', (def) => {
 	cy.visit(def);
 });
 
+Then(
+	'the user is redirected to {string} with query parameters {string}',
+	(redirectUrl, queryParams) => {
+		cy.location('href').should('include', `${redirectUrl}?${queryParams}`);
+	}
+);
+
```

Comment:
Could you move this step out of common/index.js and place it in Pagination/index.js? We are trying to avoid merge conflicts so it was decided not to touch commons

Reply:
Sure

Reply:
Done.

Reply:
It appears this was done.